src/server/publicApi.js

```diff
@@ -47,7 +46,7 @@ const runPublicApi = (gameServer) => {
 
     if (typeof ctx.request.body.model !== 'undefined') {
         // save the model in the db, not in the setupData
-        await gameServer.db.set(`${gameName}:${gameId}:model`, ctx.request.body.model);
+        await gameServer.db.setItem(`${gameId}:model`, ctx.request.body.model);
     }
```

Comment:
I've done some more research into the db connector. There isn't really any documentation on how you're supposed to use it but it seems `setItem` and `getItem` are unique to node-persist's `FlatFile` connector. 

The [base class](https://github.com/boardgameio/boardgame.io/blob/ce8ef4a16bcc420b05c5e0751b41f168352bce7d/src/server/db/base.ts#L49-L111) that the db connector implements has functions `fetch()` and `setState`/`setMetadata`

I think for compatibility maybe we should use these functions, although these functions don't allow for saving the model.
I think we either:
- extend the `FlatFile` object to have functions for `setModel` and `fetch(matchId, {model:true})` or similar
- use getItem and setItem but only for changing the model

What do you think @ChristophNiehoff

Reply:
another alternative is to just save the model somewhere else, I think the most elegant solution might be to implement setModel and fetch to save the model

Reply:
I've implemented this extending the `FlatFile` object. This means in the actual code it uses functions that will be available whatever the db connector used (if we were to change it at some point/ someone needs to use mongo)